internal/run/convert_test.go

```diff
@@ -40,3 +47,127 @@ func TestV3ReleaseFound(t *testing.T) {
 	assert.True(t, v3ReleaseFound("myapp", cfg))
 	assert.False(t, v3ReleaseFound("doesnt_exists", cfg))
 }
+
+func clientsetWithV2ConfigmapsMock() *fake.Clientset {
+	return fake.NewSimpleClientset(
+		&corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "myapp.v1",
+				Namespace: "example",
+				Labels: map[string]string{
+					"NAME":    "myapp",
+					"OWNER":   "TILLER",
+					"STATUS":  "DEPLOYED",
+					"VERSION": "1",
+				},
+			},
+		},
+		&corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "myapp.v2",
+				Namespace: "example",
+				Labels: map[string]string{
+					"NAME":    "myapp",
+					"OWNER":   "TILLER",
+					"STATUS":  "DEPLOYED",
+					"VERSION": "2",
+				},
+			},
+		},
+		&corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "other.v1",
+				Namespace: "example",
+				Labels: map[string]string{
+					"NAME":    "other",
+					"OWNER":   "TILLER",
+					"STATUS":  "DEPLOYED",
+					"VERSION": "1",
+				},
+			},
+		},
+	)
+}
+
+func TestGetV2ReleaseConfigmaps(t *testing.T) {
+
+	c := NewConvert(env.Config{Release: "myapp", TillerNS: "example"}, "", "")
+	clientset := clientsetWithV2ConfigmapsMock()
+
+	cmaps, err := c.getV2ReleaseConfigmaps(clientset)
+	assert.NoError(t, err)
+	assert.Equal(t, len(cmaps.Items), 2)
```

Comment:
You should also assert that you retrieved the correct configmaps.

Reply:
done